package cmexample1domain.databasevalues;

message FooValues {
    repeated string value = 1;
}
